config/environments/production.rb

```diff
@@ -74,21 +68,15 @@
   # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
 
-  # Send deprecation notices to registered listeners.
-  config.active_support.deprecation = :notify
-
-  # Log disallowed deprecations.
-  config.active_support.disallowed_deprecation = :log
-
-  # Tell Active Support which deprecation messages to disallow.
-  config.active_support.disallowed_deprecation_warnings = []
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
```

Comment:
does this mean we don't get deprecation warnings at all in the logs/alerts anymore?

Reply:
ah but i guess we'll still see warnings when running tests locally?  seems like it'd be nice to get deprecation warnings somewhere.

Reply:
(but also didn't seem worth holding up merge over that question)